src/index.html

```diff
@@ -11,8 +11,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute should point to a compiled CSS file (e.g., `index.css`) instead of an SCSS file. Ensure that the SCSS is compiled to CSS before linking it here.